snyk/models.py

```diff
@@ -36,11 +36,15 @@ class Vulnerability(DataClassJSONMixin):
     cvssScore: Optional[str] = None
 
 
+@dataclass
+class LicenseIssue(DataClassJSONMixin):
+    id: str
```

Comment:
I think we should add the rest of the fields, either in a quick follow up or in this PR. Otherwise we're hiding some of the data.